frontend/packages/dev-console/src/components/topology/components/nodes/PodSet.tsx

```diff
@@ -41,6 +51,14 @@ const PodSet: React.FC<PodSetProps> = ({ size, data, x = 0, y = 0 }) => {
     data.previous,
     data.isRollingOut,
   );
+  const accessAllowed = usePodScalingAccessStatus(
+    data.dc,
+    modelFor(data.dc.kind),
```

Comment:
modelFor will not work for crd based models (such knative revisions, event sources).
```suggestion
    modelFor(referenceFor(data.dc)),
```

Reply:
This was causing the error in the console.
![error](https://user-images.githubusercontent.com/9964343/70930037-5301e500-205a-11ea-84a7-679e543aa72d.png)

Reply:
Fixed in https://github.com/openshift/console/pull/3746/commits/15b341d74828aa8a5bdf6643c253548720b04a58